pyexasol/callback.py

```diff
@@ -74,33 +76,56 @@ def import_from_pandas(pipe, src, **kwargs):
     Basic example how to import from Pandas DataFrame
     Custom params for "to_csv" may be passed in **kwargs
     """
-    import packaging.version
     import pandas
 
     if not isinstance(src, pandas.DataFrame):
         raise ValueError("Data source is not pandas.DataFrame")
 
     wrapped_pipe = io.TextIOWrapper(pipe, newline="\n", encoding="utf-8")
+    src.to_csv(
+        wrapped_pipe,
+        header=False,
+        index=False,
+        lineterminator="\n",
+        quoting=csv.QUOTE_NONNUMERIC,
+        **kwargs,
+    )
+
+
+def import_from_parquet(pipe, source: Path, **kwargs):
+    """
+    Basic example how to import from pyarrow parquet file(s)
+
+    Args:
+        source:
+            Local filepath to a parquet file or set of files matching a glob pattern
+        **kwargs:
+            Custom params for "pyarrow.csv.WriteOptions"
+
+    Please note that nested or hierarchical types are not supported.
+    """
+    from pyarrow import (
+        csv,
+        parquet,
+        types,
+    )
 
-    # https://github.com/pandas-dev/pandas/pull/45302
-    if packaging.version.parse(pandas.__version__) >= packaging.version.parse("1.5.0"):
-        return src.to_csv(
-            wrapped_pipe,
-            header=False,
-            index=False,
-            lineterminator="\n",
-            quoting=csv.QUOTE_NONNUMERIC,
-            **kwargs
-        )
-    else:
-        return src.to_csv(
-            wrapped_pipe,
-            header=False,
-            index=False,
-            line_terminator="\n",
-            quoting=csv.QUOTE_NONNUMERIC,
-            **kwargs
-        )
+    if not isinstance(source, Path):
+        raise ValueError(f"source {source} is not a `pathlib.Path`")
+
+    matching_files = list(source.parent.glob(source.name))
```

Comment:
that is no how you should use glob, this is an implicit glob, I think, the more clear interface would be, Path, str where we apply glob and iterable of Path